src/test/java/mpack/Assignement2.java

```diff
@@ -0,0 +1,59 @@
+package mpack;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.WindowType;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement2 {
+	public static void main(String[] args)
+	{
+		WebDriverManager.chromedriver().setup();
+		WebDriver driver = new ChromeDriver();
+		driver.get("http://www.fb.com/");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+		String currentUrl = driver.getCurrentUrl();
+		if(currentUrl.equals("https://www.facebook.com/"))
+		{
+			System.out.println("Redirected to correct URL:" +currentUrl);
+		}
+		else
+		{
+			System.out.println("Redirected to wrong URL:" +currentUrl);
+		}
+		WebElement CreateAccount = driver.findElement(By.xpath("//a[starts-with(@id, 'u_0_0_')]"));
+		System.out.println("Create Account button visibility status:" +CreateAccount.isDisplayed());
+		CreateAccount.click();
+		driver.findElement(By.name("firstname")).sendKeys("Anvi");
```

Comment:
In future , when  you will be building a framework 
move it to the properties file
Means , any change happens in location of the element and it is getting used in multiple test cases will automatically take effect 
This reduces duplicate work